iframe-sandbox/src/ipc.ts

```diff
@@ -118,6 +125,7 @@ export enum GuestMessageType {
   Write = "write",
   Read = "read",
   LLMRequest = "llm-request",
+  WebPageRequest = 'readwebpage-request'
```

Comment:
nit: use same casing/naming for request/response: `ReadWebpageRequest`, `ReadWebpageResponse`